src/app/rundown-view/components/rundown-header/rundown-header.component.ts

```diff
@@ -6,8 +6,9 @@ import { ShowStyleVariant } from '../../../core/models/show-style-variant'
 import { SubscriptionLike } from 'rxjs'
 import { IconButton, IconButtonSize } from '../../../shared/enums/icon-button'
 import { Logger } from '../../../core/abstractions/logger.service'
+import { RundownTimingService } from '../../../core/services/rundown-timing.service'
 
-const UPDATE_LOCAL_CLOCK_INTERVAL: number = 1000
+const TIME_RESOLUTION_INTERVAL: number = 1000
```

Comment:
Is once a second enough?

Also, there is no indication whether this is miliseconds or something else

Reply:
This will be deleted in SOF-1680.